cognite/neat/_graph/loaders/_rdf2dms.py

```diff
@@ -244,7 +284,7 @@ def _create_node(
         return dm.NodeApply(
             space=self.instance_space,
             external_id=identifier,
-            type=dm.DirectRelationReference(view_id.space, type_) if type_ is not None else None,
```

Comment:
this was left-over from days when we only supported 1-1 between info and dms rules